pkg/shares/share_splitting_test.go

```diff
@@ -38,7 +38,7 @@ func TestSplitTxs_forTxShares(t *testing.T) {
 						[]byte{
 							0x1,                // info byte
 							0x0, 0x0, 0x0, 0x2, // 1 byte (unit) + 1 byte (unit length) = 2 bytes sequence length
-							0x0, 0x0, 0x0, 0x2a, // reserved bytes
+							0x0, 0x0, 0x0, 0x26, // reserved bytes
```

Comment:
[Question] Why `0x2a` is replaced with `0x26`?

Reply:
[Suggestion, non-blocking] It would be incredibly helpful for code readability to have a description of each test function, providing a clear explanation of what is being precisely tested.

Reply:
These tests are probably too low level and we're considering removing them. https://github.com/celestiaorg/celestia-app/issues/1522

They were written when we were making a lot of low level byte changes to a share but totally agreed that they're cumbersome to maintain.

Reply:
> [Question] Why 0x2a is replaced with 0x26?

Because the reserved bytes contains the index of the first transaction in the share. Since the namespace size decreased, the index had to be decreased. 

`0x2a` => `42` 
`0x26` => `38`

The namespace size decreased by 4 bytes so the index of the first transaction had to be reduced by 4.